### DISCUSSION TITLE
paper api structuretype miss the structure ancient city

### DISCUSSION THREAD

**Discussion by silverunter** [PaperMC/Paper#10807] [Ideas]
### Is your feature request related to a problem?

Ancient city can't use in paper api

### Describe the solution you'd like.

Ancient city was added in minecraft in 1.19.Now minecraft is nearly 1.21.But i still can't find ancient city in the paper api 。org.bukkit.structuretype maybe still 1.18

### Describe alternatives you've considered.

spigot bukkit have no classes to  due with structure

**Comment by TonytheMacaroni** [comment]
Most of the newer structures use the jigsaw structure type, including the ancient city.

https://jd.papermc.io/paper/1.20.6/org/bukkit/generator/structure/StructureType.html#JIGSAW
